wiki/Difficulties/en.md

```diff
@@ -13,10 +15,10 @@ Star Rating and Web Icons
 
 This is the range that decides which Web Icons are used depending on the star rating of any given map. Following the icon range is a good estimate to achieve a balanced difficulty spread. There are some guidelines you can follow to improve the difficulty spread of your mapset.
 
-[Standard](Standard) difficulty spreads
+[Standard](/wiki/Game_Modes/osu!) difficulty spreads
 --------------------------------------------------
 
-From the [Ranking Criteria](Ranking_Criteria):
+From the [Ranking Criteria](/wiki/Ranking_Criteria):
 
 "**The difficulties in the mapset must be in a consecutive order. Easy or Normal can be skipped if the gap in the star rating spread allows it. If your mapset has two difficulties, one of them cannot be an Insane or Expert. The lowest difficulty must be below 2.0 stars.**"
 
```

Comment:
line 37: change `**NOT**` to `**not**`

line 47: change `Star Rating` to `star rating`
line 47: change `Web Icons` to `web icons`